langwatch_nlp/tests/studio/test_wrapped_module_entrypoint.py

```
@@ -0,0 +1,76 @@
+"""
+
+import dspy
+import pytest
+
+from langwatch_nlp.studio.dspy.langwatch_workflow_module import (
+    LangWatchWorkflowModule,
+)
+
+
+class ForwardOnlyModule(dspy.Module):
+    def forward(self, input: str):
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)